hw6/6.2/list.c

```diff
@@ -0,0 +1,31 @@
+#include <stddef.h>
+#include <stdlib.h>
+#include "list.h"
+
+List** addNodes(int nodesNumber) {
```

Comment:
```suggestion
List** createCycledList(int nodesNumber) {
```
Если это добавление элементов, то в эту функцию должны передавать список, в который будут добавлены элементы.
А так, это создание списка